terraform/performance/main.tf

```diff
@@ -49,6 +49,7 @@ resource "null_resource" "integration_test" {
       "echo sha ${var.cwa_github_sha}",
       "cloud-init status --wait",
       "echo clone and install agent",
+      "rm -rf amazon-cloudwatch-agent-test",
```

Comment:
Instead of hardcoding the folder name, can you use `${var.github_test_repo`?

Reply:
Forgot to delete it.Yes